crates/ruma-client-api/src/uiaa/get_uiaa_fallback_page.rs

```diff
@@ -35,35 +34,179 @@ pub mod v3 {
         pub session: String,
     }
 
-    /// Response type for the `authorize_fallback` endpoint.
-    #[response(error = crate::Error)]
-    #[derive(Default)]
-    pub struct Response {
-        /// Optional URI to redirect to.
-        #[ruma_api(header = LOCATION)]
-        pub redirect_url: Option<String>,
-
-        /// HTML to return to client.
-        #[ruma_api(raw_body)]
-        pub body: Vec<u8>,
-    }
-
     impl Request {
         /// Creates a new `Request` with the given auth type and session ID.
         pub fn new(auth_type: String, session: String) -> Self {
             Self { auth_type, session }
         }
     }
 
+    /// Response type for the `authorize_fallback` endpoint.
+    #[derive(Debug, Clone)]
+    #[non_exhaustive]
```

Comment:
Shouldn't we respect the unstable exhaustive feature here?

Reply:
> Shouldn't we respect the unstable exhaustive feature here?

Doubt so as to my knowledge there isn't any additional variant that could be added to this enum in the future, but feel free to elaborate on why we would benefit from `non_exhaustive` here if there's something I'm not aware of.

Reply:
Ah yes, maybe we don't need the attribute here at all. This comment was actually meant more for the structs below.

Reply:
Done.